dataclients/kubernetes/kube.go

```diff
@@ -498,9 +502,21 @@ func (c *Client) LoadAll() ([]*eskip.Route, error) {
 	c.current = mapRoutes(r)
 	log.Debugf("all routes loaded and mapped")
 
+	go c.registerSigtermHandler()
```

Comment:
even better.. with a select, we don't even need the goroutine here. Then no need for the cleanup either. Pushing it in a bit.